templates/paragraphs/paragraph--oe-iframe-media.html.twig

```diff
@@ -0,0 +1,32 @@
+{#
+/**
+ * @file
+ * Theme override to display the 'Iframe' paragraph.
+ *
+ * @see ./modules/contrib/paragraphs/templates/paragraph.html.twig
+ */
+#}
+
+{# Allow cache from fields to bubble up since we use only values from them. #}
```

Comment:
It is approach from https://github.com/openeuropa/oe_theme/pull/789/files#diff-96c12e6bb038adadfcef1cc7c2c635be8d18e956e391efe3797981ce3952f32cR10 and https://github.com/openeuropa/oe_theme/pull/787/files#diff-e8ef914d08dc95ae7d28b49062142ef7e0181ea1dff787b83d2a5cbd5cf7a6f1R10